packages/block-library/src/page-list/block.json

```diff
@@ -51,6 +51,18 @@
 		},
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"__experimentalBorder": {
+			"radius": true,
```

Comment:
To properly support border radius, we likely need to also adopt padding and margin supports for the block.

Reply:
Resolved via https://github.com/WordPress/gutenberg/pull/66385/commits/3406a5ad551f538a8b8cb3220c9cb157451bcdaf however the spacing supports likely shouldn't be displayed by default.